.dotnet/src/Custom/VectorStores/VectorStoreClient.Protocol.cs

```diff
@@ -145,7 +145,7 @@ public virtual CreateVectorStoreOperation CreateVectorStore(BinaryContent conten
     /// <exception cref="ClientResultException"> Service returned a non-success status code. </exception>
     /// <returns> The response returned from the service. </returns>
     [EditorBrowsable(EditorBrowsableState.Never)]
```

Comment:
We should be able to remove the [EBN] if the methods are internal/

Reply:
I'm not sure what that attribute does, but it was already there :)

Reply:
removed for internal methods